docs/getting-started.md

```diff
@@ -0,0 +1,140 @@
+# Getting started
+
+To start LocoKit, you can use the docker-compose installation
+or by cloning the repository.
+
+## docker-compose installation
+
+On your host, you'll have to create a `docker-compose.yml` file.
+
+You can download it from 
+https://raw.githubusercontent.com/locokit/locokit/master/docker-compose-starter.yml
+directly if you want.
+
+```yaml
+# This docker-compose is for testing purpose only
+# For a deployment, you'll need to set a mail server
+# and security with helmet environment variables,
+# maybe a nginx for serving front and reverse proxy to the api and the file storage.
+version: '3'
+services:
+  lck-db:
+    image: postgis/postgis:12-3.0
+    environment:
+      POSTGRES_DB: public
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: pouicpouic
```

Comment:
Maybe we need to change this. It can be misinterpreted. 

If agree, we also need to update l53

Reply:
What are your thoughts ?

Reply:
Replace by `yourPostgresPassword`